circleci/templates/gradle/config_template-v0_1.yml

```diff
@@ -1,159 +1,158 @@
-#Base CircleCI template for Gradle projects.
-#Defined below are steps for building, OWasp, and Sonar
+# This configuration pipeline is meant to outline structure and naming standards
+# Author: Jonathan Fontaine
+version: 2.1 #CircleCI API Version
+orbs:
+  snyk: snyk/snyk@0.0.8
 
-version: 2.1
-jobs:
-  incremental_build:
-    docker: &image
+jobs: #Defines the overall Jobs for this particular pipeline to be used in workflows or as isolated jobs
+  Build: # Execute compilation and library linkage as first step
+    docker: &image # The '&image' setups up a reference tag to be used by '*image' (see unit tests)
       - image: circleci/openjdk:11-jdk
     steps:
-      #This sets some environment variables. Some could be moved into a CircleCI context.
-      - run: &env
+      - run: &env # does the same as '&image' above and see usage below in unit tests
           name: Setting environment
           command: |
-            echo "export RELEASE=0" >> $BASH_ENV
-            echo "export RELEASE_VERSION=NA" >> $BASH_ENV
-            echo "export RELEASE_TAG=NA" >> $BASH_ENV
-            echo "export NEXT_VERSION=NA" >> $BASH_ENV
-
-            echo "export ION_GPG_KEYRING=$ION_RELEASE_KEY" >> $BASH_ENV
             echo "export DISABLE_DOWNLOAD_PROGRESS_OPTS=-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn" >> $BASH_ENV
-            echo "export LINUX_MVN_RANDOM=-Djava.security.egd=file:/dev/./urandom" >> $BASH_ENV
-            echo "export COVERAGE_EXCLUSIONS=**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml" >> $BASH_ENV
-            echo "export GITHUB_USERNAME=connexta" >> $BASH_ENV
-            echo "export GITHUB_REPONAME=multi-int-store" >> $BASH_ENV
-            echo "export GITHUB_KEY=multi-int-store-github-key" >> $BASH_ENV
-            echo "export ORG_GRADLE_PROJECT_dependencyCheckUrlModified='http://10.101.190.7:10179/nvdcve-1.0-modified.json.gz'" >> $BASH_ENV
-            echo "export ORG_GRADLE_PROJECT_dependencyCheckUrlBase='http://10.101.190.7:10179/nvdcve-1.0-%d.json.gz'" >> $BASH_ENV
             source $BASH_ENV
       - checkout
       - run:
-          name: Building
-          command: |
-            ./gradlew build --info
-      - run:
-          name: Save test results
+          name: Maven install without tests
           command: |
-            mkdir -p ~/test-results/junit/
-            find . -type f -regex ".*/target/surefire-reports/.*xml" -exec cp {} ~/test-results/junit/ \;
-          when: always
-      - store_test_results:
-          path: ~/test-results
-      - store_artifacts:
-          path: ~/test-results/junit
-  build:
+            ./gradlew install -B -DskipTests $DISABLE_DOWNLOAD_PROGRESS_OPTS
```

Comment:
The $DISABLE_DOWNLOAD_PROGRESS_OPTS environment variable generates maven plugin logger guidance. I suspect it won't work properly for gradle.  Have you run this manually to confirm it works as expected?

The same concern exists for me around the -DskipTests command.  We should review and have some guidance around the gradle file and the different stages it allows.